crates/fuel-streams-core/src/outputs/subjects.rs

```diff
@@ -0,0 +1,310 @@
+use fuel_core_types::fuel_types::{Address, Bytes32, ContractId};
+use fuel_streams_macros::subject::{IntoSubject, Subject};
+
+use crate::types::*;
+
+/// Represents the NATS subject for all outputs.
+///
+/// This subject format allows for querying all outputs, optionally filtered by
+/// transaction ID and index.
+///
+/// # Examples
+///
+/// **Creating a subject for all outputs of a specific transaction:**
+///
+/// ```
+/// use fuel_streams_core::outputs::subjects::OutputsAllSubject;
+/// use fuel_streams_core::types::*;
+/// use fuel_streams_macros::subject::SubjectBuildable;
+///
+/// let subject = OutputsAllSubject::new()
+///     .with_tx_id(Some(Bytes32::zeroed()))
+///     .with_index(Some(0));
+/// assert_eq!(
+///     subject.to_string(),
+///     "outputs.0x0000000000000000000000000000000000000000000000000000000000000000.0.>"
+/// );
+/// ```
+#[derive(Subject, Debug, Clone, Default)]
+#[subject_wildcard = "outputs.>"]
+#[subject_format = "outputs.{tx_id}.{index}.>"]
```

Comment:
If you check how it works inside the macro, this `subject_format` is being passed for a `format!()` call, so if you do a `>` in the end, you will literally post this `>`

Reply:
Hmm? I guess you are miss confusing this here sir